index.html

```diff
@@ -44,32 +44,189 @@ <h1>
         <ul>
           <li>
             <a href="https://twitter.com/NewtonOmbese">
-              <img src="./images/twitter.svg" alt="">
+              <img src="./images/twitter.svg" alt="Twitter Icon">
             </a>
           </li>
           <li>
             <a href="https://www.linkedin.com/in/newton-ombese-570862210/">
-              <img src="./images/Linkedin icon.svg" alt="">
+              <img src="./images/Linkedin icon.svg" alt="Linkedin Icon">
             </a>
           </li>
           <li>
             <a href="https://medium.com/@newtonombese1">
-              <img src="./images/medium.svg" alt="">
+              <img src="./images/medium.svg" alt="Medium Icon">
             </a>
           </li>
           <li>
             <a href="https://github.com/NewtonOmbese">
-              <img src="./images/github.svg" alt="">
+              <img src="./images/github.svg" alt="Github Icon">
             </a>
           </li>
           <li>
             <a href="">
-              <img src="./images/angellist.svg" alt="">
+              <img src="./images/angellist.svg" alt="Angellist Icon">
             </a>
           </li>
         </ul>
       </div>
     </section>
+    <section class="projects">
+      <h2>My Recent<br> Works</h2>
+      <span class="line"></span>
+      <div class="project-cards">
+        <div class="project-card">
```

Comment:
- [ ] Hello, great working on some of the changes required. Kindly please ensure that the project section is responsive as stated in the project requirement. **Hint: You nicely implemented this for the `header section` and and `About me` content.**

<img width="611" alt="Untitled" src="https://github.com/NewtonOmbese/Newton-Portfolio/assets/103386183/551024cc-6405-4f1d-9581-34b6d81a5959">

Reply:
@OyePriscilla I made the change you requested and added the breakpoints:
- 600px
- 768px
- 1028px

The first breakpoint of 600px was to fix my background image issue
The 768px breakpoint was for this layout to both my cards and here is how the cards looked

![Screenshot from 2023-06-02 15-49-12](https://github.com/NewtonOmbese/Newton-Portfolio/assets/79839603/4c87eb44-d06d-40fa-b422-bebe425e62c6)

![Screenshot from 2023-06-02 15-49-07](https://github.com/NewtonOmbese/Newton-Portfolio/assets/79839603/20fd5cfd-4afc-4aed-839a-ef2b13f7ab19)

And finally the final breakpoint gave me this layout

![Screenshot from 2023-06-02 15-50-34](https://github.com/NewtonOmbese/Newton-Portfolio/assets/79839603/29f4dfea-77b9-4ed0-851e-d11fead327f3)

![Screenshot from 2023-06-02 15-50-37](https://github.com/NewtonOmbese/Newton-Portfolio/assets/79839603/23279caa-79d8-4625-b9f9-aa451e8c6195)